.github/workflows/ci-pipeline-arm64.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       fail-fast: false
       matrix:
         cc: [gcc, clang]
-        os: ['ubuntu_18.04', 'rocky_linux_8']
```

Comment:
Are the `ubuntu_18.04-*` Docker images futile as well now and to-be-removed?

Reply:
Yes, once we have also updated odp-dpdk. The only image which is required after this is `ubuntu_18.04-i386` as it is the last Ubuntu version supporting i386.